cfme/utils/targeted_refresh.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+"""
+class for targeted refresh tests
+"""
+
+
+class TargetedRefresh(object):
+
+    def __init__(self):
```

Comment:
**optional** use attrs to declare the class

Reply:
Done.